saleor/graphql/core/mutations.py

```diff
@@ -1173,6 +1173,8 @@ def perform_mutation(  # type: ignore[override]
         cls, _root, info: ResolveInfo, /, file
     ):
         file_data: UploadedFile = cast(UploadedFile, info.context.FILES[file])
+        if not file_data.file:
+            raise ValueError("Received an empty file.")
```

Comment:
Are you sure that ValueError is ok here? Isn't it like this will be returned as an unhandled error? Shouldn't it be ValidationError?

Reply:
Yep, this should raise ValidationError, otherwise, this will be reported as "Internal Server Error" in the GraphQL response and thrown in Sentry as an unhandled exception.